contrib/benchmarking_nd4j/src/main/java/com/example/BlasWrapper.java

```diff
@@ -0,0 +1,72 @@
+
+
+        public org.nd4j.linalg.factory.BlasWrapper wrapper = Nd4j.getBlasWrapper();
+
+        static {
+            // Only needed for mkl on RC3.8
+            //System.loadLibrary("mkl_rt");
+        }
```

Comment:
We're not going to be benchmarking RC3.8 anytime soon again. Might as well get rid of this.

Reply:
Removed.